src/Files.App/Filesystem/Cloud/Detector/CloudDetector.cs

```diff
@@ -2,21 +2,21 @@
 // Licensed under the MIT License. See the LICENSE.
 
 using Files.Shared.Cloud;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 
 namespace Files.App.Filesystem.Cloud
 {
+	/// <summary>
+	/// Provides an utility for cloud detection
+	/// </summary>
```

Comment:
```suggestion
	/// <summary>
	/// Provides an utility for cloud detection.
	/// </summary>
```

It's a silly request, but you used an ending `.` in other classes

Reply:
Fixed